controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -1262,3 +1263,104 @@ def test_identity_probe(self):
         request = Mock()
         context = Mock()
         self.servicer.Probe(request, context)
+
+
+class TestControllerServerValidateVolumeCapabilities(AbstractControllerTest):
+
+    def get_create_object_response_method(self):
+        return csi_pb2.ValidateVolumeCapabilitiesResponse
+
+    def get_create_object_method(self):

```

Comment:
why `create`? what is going on here?

Reply:
done

Reply:
for *both* abstracts

Reply:
btw why do you inherit form `AbstractControllerTest`?
if there's a good reason, then we might need to rename/split that class

Reply:
for the tests..

Reply:
got it, let's rename+split

Reply:
done